primitives/beefy/src/lib.rs

```diff
@@ -209,6 +209,83 @@ sp_api::decl_runtime_apis! {
 	}
 }
 
+#[cfg(feature = "std")]
```

Comment:
Why was this moved to primitives? Maybe it's used in the rest of the PR in runtime code and so that doesn't show here. But ideally this would be compiled for tests, since it's testing infrastructure.

Reply:
Yes, this is later used in `frame/beefy` tests as well as `client/beefy` tests - `primitives/beefy` seemed the obvious place for sharing it.